test/functional/page_objects/discover_page.js

```diff
@@ -126,6 +126,7 @@ export function DiscoverPageProvider({ getService, getPageObjects }) {
     }
 
     async clickLoadSavedSearchButton() {
+      await testSubjects.moveMouseTo('discoverOpenButton');
```

Comment:
We used to have this internally but looks like it was commented out:

<img width="867" alt="Screen Shot 2020-03-04 at 10 06 03 AM" src="https://user-images.githubusercontent.com/16563603/75892723-cc120c80-5dff-11ea-9fdd-f85e3914cf5e.png">

Reply:
I commented it out during Leadfoot -> Webdriver migration, since many tests were failing on `moveMouseTo`.
The plan was to find a reason for that issue, but the easiest way was to keep it commented out.
Now I think we need to remove it to avoid confusion.

Reply:
I saw people adding `moveMouseTo` in some test/method explicitly and it looks reasonable and easy to follow.

Reply:
@stacey-gammon would you like me to remove the commented out moveMouseTo line in this PR?

Reply:
up to you, just was pointing it out, changes LGTM!